### PR TITLE
added Postal to Restaurant

### PR DESCRIPTION
**Pull request by chanqingzhou**

### CODE REVIEW

**Comment by chanqingzhou** [comment]
#100

**Review by skpai27 — APPROVED** [review]
Well done, very well implemented. Perhaps an improvement could be to rename the listUnvisited command to something like getRecommendations, seeing that one of the purposes of the applications is to allow users to actually get a recommendation of a new / highly rated place to eat.

File: src/test/java/seedu/address/logic/commands/ListUnvisitedCommandTest.java [hunk 1]
```diff
@@ -30,12 +27,12 @@ public void setUp() {
         expectedModel.updateFilteredRestaurantList(Model.PREDICATE_SHOW_UNVISITED_RESTAURANTS);
 
     }
-
-    @Test
```
Comment (weixin-koh) [inline src/test/java/seedu/address/logic/commands/ListUnvisitedCommandTest.java:34]: Perhaps you could fix this test for the next milestone as this test is currently empty

**Review by weixin-koh — APPROVED** [review]
Well done! Do remember to write the test cases for the JsonSerializablePostalData and JsonPostalDataStorage classes in the next milestone

**Review by Wklee96 — COMMENTED** [review]
for ListUnvisitedCommand, developers might not understand what are the variables x, y, aX and aY. Perhaps you can include some comments at each steps to explain what are they and what your methods are doing

File: src/main/java/seedu/address/model/UserPrefs.java [hunk 2]
```diff
@@ -14,8 +14,8 @@
 public class UserPrefs implements ReadOnlyUserPrefs {
 
     private GuiSettings guiSettings = new GuiSettings();
-    private String name = "Annon";
-    private Path foodDiaryFilePath = Paths.get("data" , "fooddiary.json");
+    private String name = "anonymous";
+    private Path foodDiaryFilePath = Paths.get("data" , "foodDiary.json");
```
Comment (weixin-koh) [inline src/main/java/seedu/address/model/UserPrefs.java]: Is there a reason for the change in the Json file path? The capitalization in the new string might result in a missing file.

**Review by weixin-koh — DISMISSED** [review]

File: src/main/java/seedu/address/model/UserPrefs.java (hunk 2 above)
Comment (chanqingzhou) [reply src/main/java/seedu/address/model/UserPrefs.java]: Done

**Pull request merged**